modules/power-monitoring-deploying-kepler.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/installing-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-deploying-kepler_{context}"]
+= Deploying {PM-kepler}
+
+You can deploy {PM-kepler} by creating an instance of the `{PM-kepler}` custom resource definition (CRD) by using the {PM-operator}. 
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*.
+
+. On the *Create {PM-kepler}* page, ensure the *Name* is set to `kepler`.
++
+[IMPORTANT]
+====
+The name of your {PM-kepler} instance must be set to `kepler`. All other instances are ignored by the {PM-operator}.
+====
+
+. Click *Create* to create a {PM-kepler} exporter daemon set.
```

Comment:
I don't see where it talks about configuring user workload monitoring.  Without this, data will not show up in the dashboards.

Reply:
You are right about data not showing up on Dashboards if UWM is not enabled but that is separate to deployment of kepler itself and isn't a prerequisite for deploying kepler. 

see: power_monitoring/visualizing-power-monitoring-metrics.adoc which mentions that UWM should be enabled. 

@ekristova , I kind of agree with @stleerh , it is worth adding a `NOTE:` to enable User Project monitoring so that metrics exposed by kepler are captured.

Reply:
Hello @sthaha I am on the fence about this one. We have a section about "accessing PM dashboards", which has this information. I understand where the worry is coming from, but I also want to avoid the repetition here. 
Also, as you added a suggestion:
. Click *Create* to deploy {PM-kepler} and Dashboards.
instead of 
. Click *Create* to create a {PM-kepler} exporter daemon set.
That does not fit well with me because this section is only about the deployment of Kepler and we did not mention Dashboards anywhere in this section. So, I am not sure it should be added here as well. :thinking:

Reply:
Oh also, another thing I realized, please correct me if I am wrong, but do not the dashboards technically show in the dashboard section even before we deploy Kepler (aka right after the installation of the Operator)? Then the information in the deployment of Kepler section would be inaccurate.

Reply:
Oh I just checked, I am wrong about the last one :sweat_smile: But my worry is still the same.

Reply:
So maybe I would suggest something like this:
.Click *Create* to create a {PM-kepler} exporter daemon set.
+
The Kepler exporter deploys power monitoring dashboards.

Something like this would be better imo. WDYT? Also, I would still not add the information about user-workload monitoring here.

Reply:
power\_monitoring/visualizing-power-monitoring-metrics.adoc only mentions this as a prerequisite.  I don't see anywhere how you actually do this or a pointer to do this.  In any case, the main point is to avoid issues like this:  
  
sustainable-computing-io/kepler-operator#340

Reply:
The way how to do it is linked in the additional resources. The link takes user to the part of the docs that give the steps (which are outside of scope  of power monitoring documentation). I hope that helps.